admissions.html

```diff
@@ -129,9 +129,7 @@ <h1 class="text-white">Admissions</h1>
                     <div class="col-md-9 m-b30">
                         <h1 class="m-b20">Openings </h1>
                         <div class="dez-separator bg-primary"></div>
-                        <p><strong>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
-                            standard dummy text ever since the  when an unknown printer took a galley of type and scrambled it to make a 
-                            type specimen book. </strong></p>
+                        <p><strong>BECAUSE OF THE UNIQUE AUDITION COMPONENT, CCCEPA DOES NOT FOLLOW THE ADMISSIONS PROCESS USED BY THE OTHER CCSD MAGNET PROGRAMS. CCCEPA HAS A ROLLING ADMISSIONS POLICY. </strong></p>
```

Comment:
I like this, but please take Caps off. Also, change the name of the organization to Tao Studio